### ISSUE TITLE
Canvas is not expanded when artboard is resized with transform fields

### ISSUE THREAD

**Issue by ktaki** [adobe-photoshop/spaces-design#3726]
Open a new AB doc. Select the AB. 

click in H field and Shift+up arrow a few times
click in W field and Shift+up arrow a few times

RESULT: The canvas is not expanded until artboard is moved by dragging it.

![screen shot 2016-02-18 at 3 12 00 pm](https://cloud.githubusercontent.com/assets/2029597/13161542/52281398-d653-11e5-88c8-e32401497758.png)

**Comment by iwehrman** [comment]
Definitely a core bug. Back to @ktaki to file a JIRA issue.

**Comment by ktaki** [comment]
https://jira.corp.adobe.com/browse/PS-1392

**Comment by chadrolfs** [comment]
Also, if you undo after this, the artboard coordinates get off (adders not in correct location).  Separate bug?

![undo-coordsoff](https://cloud.githubusercontent.com/assets/8442631/13188216/cf54e0c4-d704-11e5-9111-e345b966e467.gif)

**Comment by ktaki** [comment]
Yes, I see very strange behavior. I will report this separately.

**Comment by ktaki** [comment]
Oh, sorry. What I thought strange was expected behavior after all. @chadrolfs What do you see after undo?

**Comment by chadrolfs** [comment]
After I undo several times, the AB adders (+) are not aligned correctly with the AB, the coordinates are off.

**Comment by ktaki** [comment]
Oh, I see. The policy frame. Yes, I've seen it a few times. I will take a look if I can find reliable repro steps. Thanks.

**Comment by iwehrman** [comment]
Over to @chadrolfs to file a separate issue for the undo issue. (May already be filed.)
